src/main/java/com/brightpath/learnify/controller/QuizController.java

```diff
@@ -36,43 +37,43 @@ public class QuizController implements QuizzesApi {
     @Override
     public ResponseEntity<QuizDetailsDto> createQuiz(QuizCreationDto quizCreationDto) {
         Optional<Quiz> quiz = quizService.createQuiz(quizCreationDto.getTitle(), quizCreationDto.getDescription(),
-                UUID.fromString(quizCreationDto.getWorkspaceId()), authorizationService.defaultUser().id());
+                quizCreationDto.getWorkspaceId(), authorizationService.defaultUser().id());
         return quiz
                 .map(quizToConvert -> ResponseEntity.status(CREATED).body(quizToConvert.convertToQuizDetailsDto()))
                 .orElseGet(() -> ResponseEntity.badRequest().build());
     }
 
     @Override
-    public ResponseEntity<QuizDetailsDto> showDetailsQuizById(String quizId) {
-        Optional<Quiz> quiz = quizService.showQuizById(UUID.fromString(quizId));
-        return quiz
-                .map(quizToConvert -> ResponseEntity.status(OK).body(quizToConvert.convertToQuizDetailsDto()))
-                .orElseGet(() -> ResponseEntity.notFound().build());
+    public ResponseEntity<QuizDetailsDto> showDetailsQuizById(UUID quizId) {
+        Quiz quiz = quizService.showQuizById(quizId);
+        return ResponseEntity
+                .status(OK)
+                .body(quiz.convertToQuizDetailsDto());
     }
 
     @Override
-    public ResponseEntity<QuizSummaryDto> showQuizById(String quizId) {
-        Optional<Quiz> quiz = quizService.showQuizById((UUID.fromString(quizId)));
-        return quiz
-                .map(quizToConvert -> ResponseEntity.status(OK).body(quizToConvert.convertToQuizSummaryDto()))
-                .orElseGet(() -> ResponseEntity.notFound().build());
+    public ResponseEntity<QuizSummaryDto> showQuizById(UUID quizId) {
+        Quiz quiz = quizService.showQuizById(quizId);
+        return ResponseEntity
```

Comment:
nitpick: Can we use ResponseEntity.ok() instead of the builder-like style to maintain consistency?

Reply:
it is like this, because ReponseEntity.created() requires URI as a parametr so i maintained consistency for this class. Is that ok or?

Reply:
Using CREATED instead OK is another thing we should change (not in scope of this PR) but here I suggested to replace `ResponseEntity.status(OK).body(...)` with `ResponseEntity.ok(...)` to maintain consistency between controllers